### PR TITLE
Initial commit for Django website.

### PR DESCRIPTION
**Pull request by clundin55**
Created barebones Django website. Added logic and files for a project root, as well as a Django app called yw_website, which will contain the main website logic. Currently only a button that does nothing is displayed at the project root.

### CODE REVIEW

**Comment by aniehuser** [comment]
What are these empty .py files for?
https://github.com/aniehuser/senior-design-group10/blob/35497b735dc4c5f8101a4967fec934958b0ac022/website/yw_root/__init__.py

https://github.com/aniehuser/senior-design-group10/blob/35497b735dc4c5f8101a4967fec934958b0ac022/website/yw_website/__init__.py
 
https://github.com/aniehuser/senior-design-group10/blob/35497b735dc4c5f8101a4967fec934958b0ac022/website/yw_root/migrations/__init__.py

If they don't do anything they should either be removed or at least have a TODO

**Comment by clundin55** [comment]
What are these empty .py files for?
The __init__.py files are required to make Python treat the directories as containing packages and are auto-generated by Django.

File: website/yw_root/urls.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.index, name='index'),
+]
```
Comment (aniehuser) [inline website/yw_root/urls.py]: Always end files with newline character 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

File: website/yw_website/urls.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import include, path
+
+urlpatterns = [
+    path('yw_root/', include('yw_root.urls')),
+    path('admin/', admin.site.urls),
+]
```
Comment (aniehuser) [inline website/yw_website/urls.py]: newline

**Review by aniehuser — CHANGES_REQUESTED** [review]
Works on my end. We should probably add a Readme that indicates the following
* interpret using python3
* start server with `python3 manage.py runserver`
* access website on local host via `http://127.0.0.1:8000/yw_root/`

Other than that looks good to me

**Comment by clundin55** [comment]
Will add a readMe in a separate commit, to keep PRs small. Fixed new lines

**Review by aniehuser — APPROVED** [review]

**Pull request merged**